SharpinitUnitTest/UnitRegistryTests.cs

```diff
@@ -4,20 +4,33 @@
 using SharpInit.Units;
 using System;
 using System.IO;
+using System.Text;
 
 namespace UnitTests.Units
 {
     [TestClass]
     public class UnitRegistryTests
     {
 
+        static string path = ".\\test.service";
+        static string contents = "[Unit]\nDescription = Notepad\n\n[Service]\nExecStart = notepad.exe";
+
         [ClassInitialize]
-        public static void TestInitialize(TestContext value)
+        public static void ClassInitialize(TestContext value)
         {
             PlatformUtilities.RegisterImplementations();
             PlatformUtilities.GetImplementation<IPlatformInitialization>().Initialize();
-            UnitRegistry.InitializeTypes();
-            UnitRegistry.ScanDefaultDirectories();
+            //UnitRegistry.InitializeTypes();
+            //UnitRegistry.ScanDefaultDirectories();
+
+            if (File.Exists(path))
+            {
+                File.Delete(path);
+            }
+            FileStream unitFileStream = File.Create(path);
```

Comment:
We should really use File.WriteAllText instead of these lines

Reply:
Done